scripts/aws/setupdb.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Export settings required to run psql non-interactively
+export PGHOST=$(cat /etc/mmw.d/env/MMW_DB_HOST)
+export PGDATABASE=$(cat /etc/mmw.d/env/MMW_DB_NAME)
+export PGUSER=$(cat /etc/mmw.d/env/MMW_DB_USER)
+export PGPASSWORD=$(cat /etc/mmw.d/env/MMW_DB_PASSWORD)
```

Comment:
I had this thought before too. I ended up siding with consist naming in the application configuration vs. conforming to the `libpq` standard.

It is nice for PostgreSQL, but then other things like Redis don't have good names.